temba/locations/tests.py

```diff
@@ -26,7 +27,8 @@ def test_boundaries(self):
         self.assertEquals(self.country, response.context['object'])
 
         # ok, now get the geometry for rwanda
-        response = self.client.get(reverse('locations.adminboundary_geometry', args=[self.country.osm_id]))
+        response = self.client.get(
+            reverse('locations.adminboundary_geometry', args=[self.country.osm_id]))
```

Comment:
Same on these, please follow the conventions already in place. We don't break until we need to, then have continuation from there. Sorry, it's a pain, but have to keep the codebase consistent or it will look bananas.